_1327/main/models.py

```diff
@@ -83,6 +78,13 @@ def can_edit(self, user):
 	def can_delete(self, user):
 		return self.can_edit(user)
 
+	@classmethod
+	def used_permissions(cls):
+		return (
+		("{app}.{codename}".format(app=ContentType.objects.get_for_model(cls).app_label, codename=cls.VIEW_PERMISSION_NAME), _('view')),
```

Comment:
could you put the content type retrieval to the top of that function?
Like: `app_label = ContentType.objects.get_for_model(cls).app_label`